client/post-editor/editor-delete-post/index.jsx

```diff
@@ -69,23 +73,21 @@ class EditorDeletePost extends React.Component {
 					this.sendToTrash();
 				}
 			},
-			this.props.translate( 'Move to trash' ),
-			this.props.translate( 'Back' )
+			translate( 'Move to trash' ),
+			translate( 'Back' )
 		);
 	};
 
 	render() {
-		const { post } = this.props;
-		if ( ! post || ! post.ID || post.status === 'trash' ) {
+		const { postId, postStatus, translate } = this.props;
+		if ( ! postId || postStatus === 'trash' ) {
 			return null;
 		}
 
 		const classes = classnames( 'editor-delete-post__button', {
 			'is-trashing': this.state.isTrashing,
 		} );
-		const label = this.state.isTrashing
-			? this.props.translate( 'Trashing...' )
-			: this.props.translate( 'Move to trash' );
+		const label = this.state.isTrashing ? translate( 'Trashing…' ) : translate( 'Move to trash' );
```

Comment:
:disappointed: 21 existing translations will be lost with this change.
If the string must change, the following option has already been translated **43** times:
`translate( 'Trashing Page' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/editor/delete-post-confirmation)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).